tensorflow-core.yaml

```diff
@@ -156,6 +161,7 @@ subpackages:
         - py${{range.key}}-opt-einsum
         - py${{range.key}}-pip
         - py${{range.key}}-protobuf
+        - py${{range.key}}-pybind11
```

Comment:
TBD if this is required, but the SIGILL errors pointed to `pybind11` so it seemed good to include at any rate.